### DISCUSSION TITLE
Suggested firmware for WyzeCam V3

### DISCUSSION THREAD

**Discussion by johnvprcam24573455623** [gtxaspec/wz_mini_hacks#546] [General]
Hi guys,

I currently have WyzeCam V3  with the Wyze RSTP Beta version 4.61.0.3.

I guess I need to disable RSTP to get wz_mini_hacks  to work 

I am wondering which version is the optimal match to wz_mini_hacks, and where I can download it from.

Thanks

John

**Comment by gtxaspec** [comment]
4.36.9.139

**Comment by Bob-AOL** [comment]
Wz_mini will run with the 4.61.0.3 RTSP beta firmware just fine but the Wyze RTSP server will prevail over the wz_mini RTSP server.

 I personally run my v3s that way and use wz_mini primarily for the USB/POE/ETH functionality.

**Reply by johnvprcam24573455623** [reply]
I am wondering if wz_mini RTSP will run on 4.61.0.3 if I disable  wize RTSP?

I'm having issue with Frtigate and they suggest to use wz_mini RTSP

**Reply by Bob-AOL** [reply]
Frigate should run either if things are configured appropriately. The Wyze RTSP Beta firmware is actually quite stable and has been working well for me straight up with Blue Iris using both USB ethernet and wi-fi for the better part of a year. Not even a hiccup.

Easy enough to toggle off RTSP in 4.61.03, though RTSP is what that particular firmware is all about in the first place. Disabling the stream may or may not disable the server, so...

If wz_mini is loading and not giving you problems in and of itself, re-flash 4.36.9.139 and see how that plays. Though if 4.61.0.3 is already providing a stream I'd be inclined to stick with it and drill a little further down into Frigate for a solution there. 

I don't think the Wyze Beta is your problem.

**Reply by teredactle** [reply]
I can't get mine to run mini hacks with the RTSP firmware. I have the latest mini_hacks installed, but the front status light just blink blue. if I take the SD card out, it will connect to wifi just fine. Is there a possible issue with the RTSP beta wyze formware and wifi? I don't have my POE dongle handy to check if this may be the reason

**Reply by Bob-AOL** [reply]
All 8 of my wz-mini V3s are running the Wyze beta RTSP 4.61.0.3 firmware. Six hardwired USB ethernet on UCTRONICS adapters and two wi-fi. All POE except one wi-fi on a USB 5V power supply. All rock solid and very smooth with nearly zero latency or dropouts.

As you may recall, a past discussion seemed to reveal the beta firmware RTSP server itself prevailed as the actual server when using the Wyze 4.61.0.3 firmware but happily coexisted with wz_mini otherwise ..

**Reply by teredactle** [reply]
Thanks Bob, def recall, and thanks for confirming. Are you on the latest version of mini-hacks? I'm asking (and my previous question too) because in the change notes I noticed a new option for RTSP go2rtc which at the time we had the discussion was not implemented, so curious whether the same holds true (in regards to what we saw back then with the RTSP on the wyze beta fw) even with this new go2rtc. Is that something you looked at?

**Reply by Bob-AOL** [reply]
I have not looked at go2rtc and I believe my current wz_mini release is circa Dec. 2022.

So you're still chasing the ever elusive high quality audio from that little v3 mic?

**Comment by teredactle** [comment]
Yup still chasing better audio. Looks like the beta wyze RTSP firmware is still preventing from another RTSP server to be used. go2rtc video is still not working via RTSP with go2rtc. Basically no video (black video) even though 1) it's on a different port, 8554 and not 554, and 2) having turned off RTSP in the wyze app and rebooted camera.
